pkg/infra/slugify/slugify.go

```diff
@@ -0,0 +1,343 @@
+/*
+ * The MIT License (MIT)
+ *
+ * This file evolved from the MIT licensed: https://github.com/machiel/slugify
+ */
+package slugify
+
+import (
+	"bytes"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/gofrs/uuid"
+)
+
+var (
+	simpleSlugger = &slugger{
+		isValidCharacter: validCharacter,
+		replaceCharacter: '-',
+		replacementMap:   getDefaultReplacements(),
+	}
+)
+
+// Slugify creates a URL safe latin slug for a given value
+func Slugify(value string) string {
+	s := simpleSlugger.Slugify(value)
+	if s == "" {
+		// If the name is only characters outside of the
+		// sluggable characters, use a content hash instead
+		s = uuid.NewV5(uuid.NamespaceOID, value).String()
```

Comment:
previously we used:
```
base64.RawURLEncoding.EncodeToString([]byte(title))
```
that gets pretty big, so i think this is better -- but I do not feel too strongly